index.html

```diff
@@ -64,7 +64,7 @@ <h1 id="title">Top Github Issues</h1>
 }
 
 function getParam(name) {
-    var match = RegExp('[?&]' + name + '=([^&]*)').exec(window.location.search);
+    var match = RegExp('[?&]' + name + '=([^&/]*)').exec(window.location.search);
```

Comment:
Huh.  When I was working on it before, I was getting an automatically-added trailing slash that messed up this parsing, but now I can't reproduce it.